### PR TITLE
[PROCEDURES] Implement policies for committers and pull requests.

### PR DESCRIPTION
**Pull request by jmchilton**
- State and clarify existing informal commit and pull request policies.
- Add a list of committers who should have "commit" access to the repository.
- Add language describing how the Galaxy project will manage commit access and repository governance going forward.

This fills in a lot of gaps left by informal policies. Much of the language and structure is borrowed with heavy modification from the Django and Apache variants of these documents.

This pull request is the consensus reached as part of the process pull request #295. In accordance with the policy - this pull request should remain open for 192 hours, must not be modified after opened, and requires at least 5 binding +1s and no binding -1s.

### CODE REVIEW

**Comment by natefoo** [comment]
+1

**Comment by jmchilton** [comment]
:+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -103,3 +102,74 @@ developers.
...
+on pull requests. Information on the *committers* group can be found in the
+[organization document](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst)
+describing governance of the core Galaxy code base.
+
+Votes on pull requests should take the form of
+[+1, 0, -1, and fractions](http://www.apache.org/foundation/voting.html)
+as outlined by the Apache Foundation.
+
+Pull requests modifying pre-existing releases should be restricted to bug fixes
+and require at least 2 *+1* binding votes from someone other than the author of
```
Comment (martenson) [inline CONTRIBUTING.md:119]: what are the fractions in the paragraph above for if we require "2 _+1_ binding votes"? Shouldn't it be  +2 total? (is +1+0.6+0.4 the same as "2 _+1_ binding votes" ?)

Comment (jmchilton) [inline CONTRIBUTING.md:119]: Just a informal shorthand language that is used to communicate ideas - to help the discussion on PRs and register partial dissent or ascent. They are not used in a formal way - I think that is fine though and they are still valuable.

File: doc/source/project/organization.rst [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+membership to the `committers` group - as such new members may be added to
+this group by opening a pull request adding a qualified person to this list.
+Pull requests modifying the membership of this list are subject to the normal
+rules for pull requests that modify governance procedures outlined in Galaxy's
+CONTRIBUTING_ document - with one exception - a `commmitter` may not vote
+against their own removal from the group (for obvious reasons).
+
+Given the responsibilities and power invested in this group - it is important
+that individuals not actively working on Galaxy in some fashion are removed
+the group. If individuals in this group intend to change jobs or reallocate
```
Comment (nsoranzo) [inline doc/source/project/organization.rst:77]: A "from" seems to be missing between `removed` and `the group`.

**Comment by jxtx** [comment]
+1.

**Comment by nsoranzo** [comment]
:+1:

**Comment by martenson** [comment]
:+1:

**Comment by bgruening** [comment]
:+1:

**Comment by hexylena** [comment]
:+1:

**Comment by dannon** [comment]
+1

**Comment by bgruening** [comment]
The required time has elapsed and I'm going to merge this. From today @erasche, @nsoranzo and /me have official commit access to this repository. 

Thank you so much for the trusting us! 
I think this is a very good sign for the Galaxy community in general and I can only encourage everyone to join this awesome community!

@erasche @nsoranzo now is time to celebrate a little bit before starting new projects!

**Pull request merged**